### PR TITLE
Translate /README.md

### PR DESCRIPTION
**Pull request by ThomasJanUta**
## Description

> README.md
> ...
> (Feel free to translate this document and add any content you feel would be useful to contributors)

I took inspiration from the [spanish repository](https://github.com/gatsbyjs/gatsby-es) that contains 469 commits already and translated the README.md page. Some sub pages do not exist yet that exist in the spanish version so I left these out.

The motivation is to make the repository more welcoming for native speakers. 😊🎉

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
+* [Gatsby-Translation-Guide](https://www.gatsbyjs.org/contributing/gatsby-docs-translation-guide/)
 * [Gatsby Discord](https://gatsby.dev/discord)
+* [Swag für Mitwirkende ✨](https://www.gatsbyjs.org/contributing/contributor-swag/)
 
-(Feel free to translate this document and add any content you feel would be useful to contributors).
-  
+
+## Wie kann ich anfangen? 🤔
+
+Beginne mit der [Translation Progress Issue](https://github.com/gatsbyjs/gatsby-de/issues/1) und melde dich für eine Unterseite, die du übersetzen möchtest. Der [Style-Guide](/style-guide.md) hilft dir bei den ersten Schritte und beantwortet die ersten Fragen. 💅
```
Comment (jonasgrunert) [inline README.md]: Same as above. Mabe better use 'dem' for Issue?

File: README.md [hunk 2]
```diff
@@ -1,15 +1,18 @@
 
-# Gatsby German Translation
+# Gatsby auf Deutsch
 
-This repo contains the German translation for Gatsby.
+Dieses Repository enthält die offizielle deutsche Übersetzung für Gatsby.
 
-Please refer to the [Translation Progress Issue](https://github.com/gatsbyjs/gatsby-de/issues/1) to start translating!
+Beginne mit der [Translation-Progress-Issue](https://github.com/gatsbyjs/gatsby-de/issues/1), um mit dem Übersetzen zu starten!
```
[the remark this answers is not in the document]
Comment (ThomasJanUta) [reply README.md]: Changed. I opened an issue regarding grammatical genders since thil will be a recurring problem #9 .

File: README.md (hunk 1 above)
Comment (ThomasJanUta) [reply README.md]: Changed.

**Comment by muescha** [comment]
Maybe also switch in PR comments always to German comments. this also in the spanish repo

File: README.md [hunk 3]
```diff
@@ -1,15 +1,18 @@
...
+* [Gatsby-Translation-Guide](https://www.gatsbyjs.org/contributing/gatsby-docs-translation-guide/)
 * [Gatsby Discord](https://gatsby.dev/discord)
+* [Swag für Mitwirkende ✨](https://www.gatsbyjs.org/contributing/contributor-swag/)
 
-(Feel free to translate this document and add any content you feel would be useful to contributors).
-  
+
+## Wie kann ich anfangen? 🤔
+
+Beginne mit dem [Translation-Progress-Issue](https://github.com/gatsbyjs/gatsby-de/issues/1) und melde dich für eine Unterseite, die du übersetzen möchtest. Der [Style-Guide](/style-guide.md) hilft dir bei den ersten Schritte und beantwortet die ersten Fragen. 💅
```
Comment (medusalix) [inline README.md]: hilft dir bei den ersten Schritte**n**...

Comment (ThomasJanUta) [reply README.md]: Habe ich verbessert =)

**Pull request merged**